### PR TITLE
Docs bugfixes

### PR DESCRIPTION
**Pull request by Vic-Cooper**
A variety of fix backports from quality week.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md [hunk 1]
```diff
@@ -2,13 +2,24 @@
 
 The following process uses DaVinci Resolve, but you can apply it for other digital content creation tools such as [Photoshop](LUT-Authoring-Photoshop.md) or Nuke.
 
-## Step 1: Set up the HDRP Asset
+To author lookup textures in resolve:
```
Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md:5]:
```suggestion
To author lookup textures in resolve: 
```
```suggestion
To author lookup textures in Resolve: 
```

File: com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md [hunk 2]
```diff
@@ -2,13 +2,24 @@
 
 The following process uses DaVinci Resolve, but you can apply it for other digital content creation tools such as [Photoshop](LUT-Authoring-Photoshop.md) or Nuke.
 
-## Step 1: Set up the HDRP Asset
+To author lookup textures in resolve: 
+
+1. [Set up the HDRP Asset](#set-up-hdrp-asset)
```
Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md:7]: I think each of these bullet items needs a full stop at the end.

Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md:7]: I think each of these bullet items needs a full stop at the end.

File: com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md [hunk 3]
```diff
@@ -25,40 +36,46 @@ After you export the EXR file, transform the data from the format that Unity use
...
 2. Open **File > Project Settings** and go to the **Color Management** panel.
 3. In the **Lookup Table** section, click the **Open Lut Folder** button. This opens your platform's file explorer at the location that stores LUTs for Resolve.
-4. In the file explorer, create a new **Unity** folder and copy all the previously imported LUTs you exported from Unity (see **Step 2)** into it.
+4. In the file explorer, create a new **Unity** folder and copy all the previously imported LUTs you exported from Unity (see [Render the Camera view to an EXR file](#camera-to-exr)) into it.
 5. Back in Resolve, click the **Update Lists** button.
-6. Set the **3D Video Monitor Lookup Table** to **Linear to sRGB r1**.
+6. Set the **Video Monitor Lookup Table** to **Linear to sRGB r1**.
 
-**Note**: You only need to do step 1-5 once for Resolve in general, and step 6 per-project.
+**Note**: You only need to do step 1 to 5 once for Resolve in general, and step 6 for each project.
```
Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md:51]:
```suggestion
**Note**: You only need to do steps 1 to 5 once for Resolve in general, and step 6 for each project.
```
```suggestion
**Note**: You only need to do step 1 to 5 once for Resolve in general, and step 6 for each project.
```

Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md:51]:
```suggestion
**Note**: You only need to do steps 1 to 5 once for Resolve in general, and step 6 for each project.
```
```suggestion
**Note**: You only need to do step 1 to 5 once for Resolve in general, and step 6 for each project.
```

File: com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md [hunk 4]
```diff
@@ -7,13 +7,13 @@ Under the hood, the Hair shader is a pre-configured Shader Graph. To learn more
...
-
-1. Go to **Windows > Package Manager**, and select **High Definition RP** from the package list.
-2. In the main window that shows the package's details, find the **Samples** section.
-3. To import a Sample into your Project, click the **Import into Project** button. This creates a **Samples** folder in your Project and imports the Sample you selected into it. This is also where Unity imports any future Samples into.
-4. In the Asset window, go to **Samples > High Definition RP > 11.0** and open the Hair scene. Here you will see the hair sample material set up in-context with a scene, and available for you to use.
+HDRP includes the Hair sample scene to help you get started. To find this Sample:
 
+1. Go to **Window** > **Package Manager**, and select **High Definition RP** from the package list.
+2. In the main window that shows the package's details, open the **Samples** drop down.
+3. Find **Material Samples** and select the **Import** button. This creates a **Samples** folder in your Project and imports the Sample you selected into it. This is also where Unity imports any future Samples into.
```
Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md:14]:
```suggestion
3. Find **Material Samples** and select **Import**. This creates a **Samples** folder in your Project and imports the Sample you selected into it. This is also where Unity imports any future Samples into.
```

File: com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md [hunk 5]
```diff
@@ -7,13 +7,13 @@ Under the hood, the Hair shader is a pre-configured Shader Graph. To learn more
...
+4. In the Asset window, go to **Samples** > **High Definition RP** and select the folder that corresponds with your HDRP version. 
+5. Open **Material Samples** > **Scenes** and select the the **Hair** scene. Here you will see the hair sample material set up in-context with a scene, and available for you to use.
```
Comment (markg-unity) [inline com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md:16]:
```suggestion
5. Open **Material Samples** > **Scenes** and select the **Hair** scene. Here you will see the hair sample material set up in-context with a scene, and available for you to use.
```

Comment (markg-unity) [reply com.unity.render-pipelines.high-definition/Documentation~/hair-shader.md:16]: I realise you didn't change this vullet but I couldn't resist removing the extra 'the'!

**Review by markg-unity — COMMENTED** [review]
Looks great! Only a handful of super-minor suggestions from me.

File: com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md (hunk 2 above)
Comment (Vic-Cooper) [reply com.unity.render-pipelines.high-definition/Documentation~/LUT-Authoring-Resolve.md:7]: Good point, thanks!

**Review by markg-unity — APPROVED** [review]
👍

**Pull request merged**